src/pe/import.rs

```diff
@@ -345,20 +345,32 @@ impl<'a> ImportData<'a> {
             "import_directory_table_rva {:#x}",
             import_directory_table_rva
         );
-        let offset =
-            &mut utils::find_offset(import_directory_table_rva, sections, file_alignment, opts)
-                .ok_or_else(|| {
-                    error::Error::Malformed(format!(
+
+        let offset_opt = utils::find_offset(import_directory_table_rva, sections, file_alignment, opts);
+
+        if offset_opt.is_none() {
+            debug!(
                 "Cannot create ImportData; cannot map import_directory_table_rva {:#x} into offset",
                 import_directory_table_rva
-            ))
-                })?;
+            );
+            return Ok(ImportData { import_data: Vec::new() });
```

Comment:
For somewhat similar reasons I don’t see why this logic has to change from failing when the import data is bad to returning an empty vector (which is like an empty string). If this is super important to make this fallible the parse options has a strict mode, or perhaps can convince me why this needs to change from being strict to doing this, but just returning empty vector because something is malformed generally not advised